Lombiq.Tests.UI/Extensions/OrchardCoreDashboardUITestContextExtensions.cs

```diff
@@ -117,4 +143,22 @@ public static Task GoToContentItemEditorByIdAsync(this UITestContext context, st
 
     public static Task GoToContentItemDisplayByIdAsync(this UITestContext context, string contentItemId) =>
         context.GoToAdminRelativeUrlAsync($"/Contents/ContentItems/{contentItemId}/Display");
+
+    /// <summary>
+    /// Clicks the "Toggle" option in the "Bulk actions" dropdown, found in the Features page.
+    /// </summary>
+    public static Task BulkActionsToggleAsync(this UITestContext context) =>
+        context.SelectFromBootstrapDropdownReliablyAsync(By.Id("bulk-action-menu-button"), "Toggle");
+
+    /// <summary>
+    /// Navigate to the Registration page.
+    /// </summary>
+    public static Task GoToRegistrationAsync(this UITestContext context) =>
+        context.GoToRelativeUrlAsync('/' + UserRegistrationParameters.DefaultUrl);
+
+    /// <summary>
+    /// Navigate to the Login page.
+    /// </summary>
+    public static Task GoToLoginAsync(this UITestContext context) =>
+        context.GoToRelativeUrlAsync("/Login");
```

Comment:
These two aren't really for the dashboard and would be more appropriate in `NavigationUITestContextExtensions`.